experiments/service.go

```diff
@@ -73,12 +72,8 @@ func (ec *experimentContext) CreateService(serviceName string, template v1alpha1
 			Annotations: serviceAnnotations,
 		},
 		Spec: corev1.ServiceSpec{
-			Selector: selector,
-			Ports: []corev1.ServicePort{{
-				Protocol:   "TCP",
-				Port:       int32(80),
-				TargetPort: intstr.FromInt(8080),
-			}},
+			Selector: template.Service.Selector,
+			Ports:    template.Service.Ports,
```

Comment:
Why are we not setting the Spec whole templates.Service here?

Reply:
Exactly, I couldn’t figure out a way to do it. Happy to learn